test/augmentation/test_auto_operation.py

```diff
@@ -14,6 +13,7 @@
 from kornia.augmentation.container import AugmentationSequential
 from kornia.geometry.bbox import bbox_to_mask
 from kornia.testing import assert_close
+from test.augmentation.test_container import reproducibility_test
```

Comment:
+1 for moving it from the test directory and adding an outside directory/module `testing` -- I would also remove the `testing` module from `kornia` since this should be just used for our test suite (I don't see the need to be distributed with the package at least)

Reply:
I introduced similar to torch.testing — eventually we can have a small separated repo I’m pypi but this increases a bit the complexity of maintenance

Reply:
put it in a separate repo does not seem to be a good idea for maintainability and also to be update/used on the same PR on kornia

Reply:
sounds good